packages/cli/src/lib/implementation/model.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
A more future-proof way is to disable a specific block, instead of the entire file.

![image](https://github.com/code-pushup/cli/assets/34691111/bf672dd6-7924-4c38-98ef-568e3a7d73e9)

Reply:
Good point! ty